automated/linux/imagenet-resnet50/imagenet-resnet50.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+TEST_GIT_URL="https://github.com/mlcommons/inference.git"
+TEST_DIR="$(pwd)/${TEST_PROGRAM}"
+TEST_PROG_VERSION="215c057fc6690a47f3f66c72c076a8f73d66cb12"
+TEST_PROGRAM="inference"
+
+usage() {
+    echo "Usage: $0 [-a <home-directory>]
+                    [-m <mount-directory>]
+                    [-t <true|false>]
+                    [-p <test-dir>]
+                    [-v <test-prog-version>]
+                    [-u <test-git-url>]
+                    [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "a:m:t:s:p:v:u:" o; do
+    case "$o" in
+        a) HOME_DIR="${OPTARG}";;
+        m) MNT_DIR="${OPTARG}" ;;
+        t) MNT_EXISTS="${OPTARG}" ;;
+        s) SKIP_INSTALL="${OPTARG}" ;;
+        p) TEST_DIR="${OPTARG}" ;;
+        v) TEST_PROG_VERSION="${OPTARG}" ;;
+        u) TEST_GIT_URL="${OPTARG}" ;;
+        *) usage ;;
+    esac
+done
+
+pkgs="build-essential git python3-venv python3-dev libhdf5-dev gcc-aarch64-linux-gnu curl"
```

Comment:
gcc-aarch64-linux-gnu, isn't that what you find when you search on a x86 machine, I don't find that on my aarch64 machine.